src/components/Image/index.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {Image as RNImage} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import lodashGet from 'lodash/get';
+import PropTypes from 'prop-types';
+import ONYXKEYS from '../../ONYXKEYS';
+import {defaultProps, imagePropTypes} from './imagePropTypes';
+import RESIZE_MODES from './resizeModes';
+
+const propTypes = {
+    ...imagePropTypes,
+
+    /** The URI source of the image */
+    source: PropTypes.shape({
+        uri: PropTypes.string.isRequired,
+    }).isRequired,
```

Comment:
I believe this is duplicated in the `imagePropTypes` file so we can remove this?

Or are you overwriting it since we should only ever pass a source with `uri` here, where on mobile we can pass a source with `uri` & `headers` OR a source that's just a `number`?

Reply:
Yes this was just a duplicate - removed it :)